data/map.txt

```diff
@@ -26871,6 +26871,12 @@ planet "Leviathan Station"
 		threshold 3500
 		fleet "Small Deep Security" 27
 
+planet "Letis'sei"
+	attributes ka'het
+	landscape land/loc3
+	description `Letis'sei was disabled long ago, but it seems the destruction it suffered was relatively small: with only a tiny fraction of the station designed to be inhabited, it didn't take much for the Ka'het to blow the crew quarters up. Today, all the air once inside the station is gone, and few of the original rooms can still be recognised.`
+	description `	Outside, three impressive, cannon-like structures slowly rotate around the living section. Each of them is more than twice as big as the crew quarters, and despite their age they still seem to be mostly intact.`
```

Comment:
twice as large*? native speakers, please confirm.

Reply:
Twice as big works fine ~~but that `is` needs to be an `are`~~

Reply:
`Big` is fine but `large` might also sound more professional.

Reply:
Fixed